### ISSUE TITLE
Incorrect description for logbook.write.max-body-size system property

### ISSUE THREAD

**Issue by goldyliang** [zalando/logbook#1329]
## Description

In README.md, it says:

logbook.write.max-body-size | Truncates the body up to max-body-size and appends .... | -1 (disabled)

But after tracing why setting '-1' does not work, I found the behavior is that, when setting to '-1', it uses a default max-body-size which is 1024:

` @Bean

  public BodyFilter bodyFilter() {

    final LogbookProperties.Write write = properties.getWrite();

    int maxBodySize = write.getMaxBodySize();

    if (maxBodySize < 0) {

      maxBodySize = DEFAULT_MAX_BODY_SIZE;

    }

    return merge(

        truncate(maxBodySize),

        merge(defaultValue(), replaceJsonStringProperty(obfuscateKeys, "XXX")));

  }
`

## Expected Behavior

If the behavior is expected, the README.md needs to be updated.

Or, the behavior shall be: if set to -1, there is no limit at all

## Actual Behavior

Setting to -1, the default limit applies, which is not as same as written in README.md

## Possible Fix

Change the code, or change README.md

## Steps to Reproduce

N/A

## Your Environment

* Version used:
* Link to your project:

**Comment by kasmarian** [comment]
Hi @goldyliang, thank you for your contribution. Could you please help me understand the problem a bit better? Looking at the current [LogbookAutoConfiguration](https://github.com/zalando/logbook/blob/94a7a7e8301f3aa9d081012b662efdd6b5c8e95b/logbook-spring-boot-autoconfigure/src/main/java/org/zalando/logbook/autoconfigure/LogbookAutoConfiguration.java#L195-L197) in the main branch, it looks like the body size filter is not applied when the value of `logbook.write.max-body-size` is `<0`. This corresponds to the README description.

```
@API(status = INTERNAL)
    @Bean
    @ConditionalOnMissingBean(BodyFilter.class)
    public BodyFilter bodyFilter() {
        final LogbookProperties.Write write = properties.getWrite();
        final int maxBodySize = write.getMaxBodySize();

        if (maxBodySize < 0) {
            return defaultValue();
        }

        return BodyFilter.merge(defaultValue(), truncate(maxBodySize));
    }
```

**Comment by goldyliang** [comment]
When looked back this issue after a long time, I realized that I was not wise of not giving out the exact version or exact link of the code I quoted, and I can not find the same piece of code again in the whole internet. I tried searching the code in 1.x, 2.x and 3.x, but could not find even the definition of "DEFAULT_MAX_BODY_SIZE".

As the latest version is correct and conforms with README, this issue can be closed.
